node/sc/bridge_manager.go

```diff
@@ -194,6 +194,7 @@ func (bi *BridgeInfo) GetCounterPartToken(token common.Address) common.Address {
 
 func (bi *BridgeInfo) GetPendingRequestEvents() []IRequestValueTransferEvent {
 	ready := bi.pendingRequestEvent.Pop(maxPendingNonceDiff / 2)
+
```

Comment:
This change seems not related to the PR's changes

Reply:
Thanks. I'll apply.